### ISSUE TITLE
Support animated GIF files.

### ISSUE THREAD

**Issue by ZacWalk** [diffractor/diffractor#9]
Currently, animated GIF files are not animated when shown. Please add support for this.

**Comment by Sethycakes** [comment]
I agree! Would this be difficult to implement? Local gif viewing is a bit cumbersome, so having Diffractor be able to handle it would make things a lot more convenient.
